resultstore-search-api/credentials.py

```diff
@@ -37,9 +36,10 @@ def __init__(self):
         self.project_id = os.environ.get('PROJECT_ID')
         self.client_id = os.environ.get('CLIENT_ID')
```

Comment:
Not sure if it's completely necessary to have this set as an environment variable. The credentials file contains the CLIENT_ID.
What's the format that the CLIENT_ID supposed to be in?

Reply:
This CLIENT_ID is used in the ui and is separate from the service account CLIENT_ID, this is because redirect uri cannot be modified for the service json.

Reply:
ok makes sense. Just wanted to check if there was a way to decrease the number of configuration variables.